plugins/org.jkiss.dbeaver.model.sql/src/org/jkiss/dbeaver/model/sql/parser/rules/ScriptParameterRule.java

```diff
@@ -28,7 +28,7 @@
 */
 public class ScriptParameterRule implements TPRule {
     
-    private final static char QUOTE_CHAR = '"';
+    private static final char QUOTE_CHAR = '"';
```

Comment:
I'm not sure if it will bring be beneficial rather than problematic, but you can use `SQLDialect#getIdentifierQuoteStrings` and `SQLDialect#getStringQuoteStrings` to build a set of quotes you might want to support.

Reply:
I added usage of `SQLDialect#getIdentifierQuoteStrings`.